src/main/scala/latis/service/hapi/AboutService.scala

```diff
@@ -6,20 +6,25 @@ import org.http4s.HttpRoutes
 import org.http4s.circe._
 import org.http4s.dsl.Http4sDsl
 
+import latis.util.LatisConfig
+import latis.util.LatisException
+
 /** Implements the `/about` endpoint. */
 class AboutService[F[_]: Concurrent] extends Http4sDsl[F] {
 
   val service: HttpRoutes[F] =
     HttpRoutes.of[F] {
       case GET -> Root / "about" =>
         Ok(
-          //TODO: add optional params?
           AboutResponse(
             HapiService.version,
             Status.`1200`,
-            "LASP",
-            "LASP HAPI Server",
-            "web.support@lasp.colorado.edu" //TODO: get from config, don't hardcode this
+            LatisConfig.get("latis.about.id").getOrElse(throw LatisException("No 'id' configured")),
+            LatisConfig.get("latis.about.title").getOrElse(throw LatisException("No 'title' configured")),
+            LatisConfig.get("latis.about.contact").getOrElse(throw LatisException("No 'contact' configured")),
```

Comment:
I think the exceptions would be a little more helpful with the full name of the property.

Reply:
Done.